dpe/src/commands/derive_context.rs

```diff
@@ -284,7 +288,39 @@ impl CommandExecution for DeriveContextCmd {
                 }
             }
         } else if self.creates_certificate() && self.exports_cdi() {
-            todo!("(clundin): Generate ECA certificiate / export random CDI")
+            cfg_if! {
+                if #[cfg(not(feature = "disable_export_cdi"))] {
+                    let mut exported_cdi_handle = [0; MAX_EXPORTED_CDI_SIZE];
+                    env.crypto
+                        .rand_bytes(&mut exported_cdi_handle)
+                        .map_err(DpeErrorCode::Crypto)?;
```

Comment:
Once refactored this will happen in the call to `create_exported_dpe_cert` and return in `CreateDpeCertResult`